Source/WebCore/editing/CompositeEditCommand.cpp

```diff
@@ -790,7 +791,7 @@ Position CompositeEditCommand::replaceSelectedTextInNode(const String& text)
 {
     Position start = endingSelection().start();
     Position end = endingSelection().end();
-    if (start.containerNode() != end.containerNode() || !start.containerNode()->isTextNode() || isTabSpanTextNode(start.containerNode()))
+    if (start.containerNode() != end.containerNode() || !start.containerNode()->isTextNode() || parentTabSpanNode(start.containerNode()))
```

Comment:
I don't understand why eliding the `!start.containerNode()->isTextNode()` check here results in crashes. After all `parentTabSpanNode()` has a `is<Text>` check.